homeassistant/components/vesync/switch.py

```diff
@@ -106,3 +110,57 @@ def __init__(self, switch):
         """Initialize Light Switch device class."""
         super().__init__(switch)
         self.switch = switch
+
+
+class VeSyncHumidifierDisplayHA(VeSyncDevice, SwitchEntity):
+    """Representation of the display on a VeSync humidifier."""
+
+    @property
+    def unique_id(self):
+        """Return the ID of this display."""
+        return f"{super().unique_id}.display"
+
+    @property
```

Comment:
Mark entity as `config` (see docs under `entity_category` [here](https://developers.home-assistant.io/docs/core/entity#generic-properties))

```suggestion
    @property
    def entity_category(self):
        """Return the config entity category."""
        return EntityCategory.CONFIG
    
    @property
```